[2025-06-05T21:21:15.041+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deep_models_pipeline.validate_models manual__2025-06-05T21:15:55.562459+00:00 [queued]>
[2025-06-05T21:21:15.047+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deep_models_pipeline.validate_models manual__2025-06-05T21:15:55.562459+00:00 [queued]>
[2025-06-05T21:21:15.048+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-06-05T21:21:15.057+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): validate_models> on 2025-06-05 21:15:55.562459+00:00
[2025-06-05T21:21:15.064+0000] {standard_task_runner.py:60} INFO - Started process 4710 to run task
[2025-06-05T21:21:15.066+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'deep_models_pipeline', 'validate_models', 'manual__2025-06-05T21:15:55.562459+00:00', '--job-id', '2024', '--raw', '--subdir', 'DAGS_FOLDER/train_deep_model_dag.py', '--cfg-path', '/tmp/tmpk88mtl95']
[2025-06-05T21:21:15.067+0000] {standard_task_runner.py:88} INFO - Job 2024: Subtask validate_models
[2025-06-05T21:21:15.096+0000] {task_command.py:423} INFO - Running <TaskInstance: deep_models_pipeline.validate_models manual__2025-06-05T21:15:55.562459+00:00 [running]> on host 3f2696f940ac
[2025-06-05T21:21:15.150+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deep_models_pipeline' AIRFLOW_CTX_TASK_ID='validate_models' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T21:15:55.562459+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-05T21:15:55.562459+00:00'
[2025-06-05T21:21:15.151+0000] {train_deep_model_dag.py:13} INFO - 🟦 Running command: python /opt/***/src/models/validate_model.py --pipeline_type=dl --test_user_count=100
[2025-06-05T21:22:06.635+0000] {train_deep_model_dag.py:20} INFO - 🟩 [stdout]:
🏃 View run movie_recommendation_validation_dl_2025-06-05_21-21-17 at: https://dagshub.com/sacer11/MLOps_movie_rec_apr25.mlflow/#/experiments/3/runs/cfa6896622364a4faf0c27a0200df06d
🧪 View experiment at: https://dagshub.com/sacer11/MLOps_movie_rec_apr25.mlflow/#/experiments/3

[2025-06-05T21:22:06.636+0000] {train_deep_model_dag.py:22} WARNING - 🟨 [stderr]:
2025-06-05 21:21:17,064 - INFO - 🚀 Starte Validierung (DL)
2025-06-05 21:21:24,663 - INFO - 📥 Daten & Modelle geladen – Beginne Evaluation
2025-06-05 21:21:38,373 - WARNING - ⚠️ Fehler bei User 51: 51
2025-06-05 21:21:48,664 - WARNING - ⚠️ Fehler bei User 91: 91
2025-06-05 21:21:50,930 - INFO - 📊 precision_10_hybrid: 0.21
2025-06-05 21:21:50,930 - INFO - 📊 precision_10_user:   0.05
2025-06-05 21:22:00,193 - INFO - 🏆 Neuer Bestwert für hybrid_deep_model! 0.214 > 0.000
2025-06-05 21:22:01,210 - ERROR - Keine Registry-Version zu diesem Run gefunden!
2025-06-05 21:22:03,173 - INFO - 🏆 Neuer Bestwert für user_deep_model! 0.051 > 0.000
2025-06-05 21:22:04,214 - ERROR - Keine Registry-Version zu diesem Run gefunden!
2025-06-05 21:22:06,203 - INFO - 🎉 Validation abgeschlossen.

[2025-06-05T21:22:06.637+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-05T21:22:06.646+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=deep_models_pipeline, task_id=validate_models, execution_date=20250605T211555, start_date=20250605T212115, end_date=20250605T212206
[2025-06-05T21:22:06.692+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-05T21:22:06.709+0000] {taskinstance.py:3280} INFO - 2 downstream tasks scheduled from follow-on schedule check
